crates/holochain/Cargo.toml

```diff
@@ -58,9 +58,10 @@ toml = "0.5.6"
 tracing = "=0.1.18"
 tracing-futures = "0.2.4"
 url = "1.7.2"
-url2 = "0.0.5"
+url2 = "0.0.6"
 url_serde = "0.2.0"
 uuid = { version = "0.7", features = [ "serde", "v4" ] }
+kitsune_p2p = { version = "0.0.1", path = "../kitsune_p2p/kitsune_p2p" }
```

Comment:
I believe kitsune_p2p is re-exported from holochain_p2p if we don't want to have to manage the specific versions here...

Reply:
@neonphog it wasn't but it is now :)